torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -712,7 +712,45 @@ def batch_norm_inference(input, weight, bias, mean, var, momentum, eps):
                 else:
                     dim0 = args[1]
                     dim1 = args[2]
-                return g.call_function(ttnn.transpose, args=(args[0], dim0, dim1))
+
+                dim0 = (dim0 + rank) % rank
+                dim1 = (dim1 + rank) % rank
+
+                # directly transpose only support rank <= 4
+                if rank <= 4:
+                    return g.call_function(ttnn.transpose, args=(args[0], dim0, dim1))
+
+                # Try to reshape,
```

Comment:
This seems to be a workaround. Do we have an issue number can be documented here?

Reply:
I leave a issue to track rank > 4 `ttnn.transpose` issue. https://github.com/tenstorrent/pytorch2.0_ttnn/issues/585,